README.md

```diff
@@ -105,11 +109,11 @@ We’re focused on helping data scientists and teams that develop AI/ML models.
 
 [**Set up a time to chat!**](https://forms.gle/9HDBHX5Y3fzuDCDAA)
 
----
+<!-- ---
 
 Before envd             |  After envd
 :-------------------------:|:-------------------------:
-![](./docs/images/before.svg)  |  ![](./docs/images/after.svg)
+![](./docs/images/before.svg)  |  ![](./docs/images/after.svg) -->
```

Comment:
This part is not commented out after rendering. Is this expected?

Reply:
Yep, I think it is unnecessary, WDYT

Reply:
Looks good to me. Github markdown preview confused me